lua/octo/pickers/telescope/previewers.lua

```diff
@@ -38,6 +38,17 @@ local issue = defaulter(function(opts)
               elseif entry.kind == "pull_request" then
                 obj = result.data.repository.pullRequest
               end
+
+              -- We have to create a new buffer since writers.write_state relies on
+              -- octo_buffers containing a buffer at a index of the current buffer number.
+              -- OctoBuffer:new adds that buffer to octo_buffers
+              OctoBuffer:new {
+                bufnr = bufnr,
+                number = obj.number,
+                repo = entry.repo,
+                node = obj,
+              }
+
```

Comment:
Wondering if it would be better to make `write_state` take an optional `isDraft` boolean to avoid creating (potentially) hundreds of OctoBuffers. Alternatively we could show if the PR is in draft or not, in a new Telescope/fzf-lua column (eg: https://github.com/pwntester/octo.nvim/blob/master/lua/octo/pickers/telescope/entry_maker.lua#L8-L41) so it does not need to be showed in the preview buffer. Thoughts?

Reply:
It would definetly be better to show if PRs are drafts in a picker column. That would turn this bugfix PR into a feature PR though but thats maybe okay

Reply:
A new one would be better. Thanks!